modules/core/src/TCModel.ts

```diff
@@ -21,7 +21,7 @@ export class TCModel extends Cloneable<TCModel> {
   private publisherCountryCode_ = 'AA';
   private version_ = 2;
   private consentScreen_: StringOrNumber = 0;
-  private policyVersion_: StringOrNumber = 4;
+  private policyVersion_: StringOrNumber = 5;
```

Comment:
I understand this is a hardcoded default but does it get overridden by the GVL embedded in the Web SDK?

Reply:
It seems not (I tested setting the GLV version to 10)
![Capture d’écran 2024-07-12 à 12 27 26](https://github.com/user-attachments/assets/3c0ea8d2-e512-4777-bcb4-e0b5cf97a98d)
Screenshot after instancing `TCModel` here https://gitlab.com/didomi/sdks/web/core/-/blob/master/src/sdk/regulations/gdpr/tcf/v2.2/tcf.service.js?ref_type=heads#L479

Reply:
When calling `__tcfapi` however, the `policyVersion` does get overridden
![Capture d’écran 2024-07-12 à 12 51 45](https://github.com/user-attachments/assets/4bd57f61-e78f-498a-b1d5-41693bfdfe70)

And it's because of these two lines:
https://github.com/didomi/iabtcf-es/blob/master/modules/cmpapi/src/CmpApi.ts#L107
https://github.com/didomi/iabtcf-es/blob/master/modules/core/src/TCModel.ts#L381

Reply:
Sounds good, thanks!